docs/onboarding.md

```diff
@@ -43,3 +43,18 @@ Then add your SSH key _also_ as the commit signing key:
 1. Open [New SSH Key](https://github.com/settings/ssh/new) page.
 2. Select `Key type`: `Signing key`.
 3. Copy the content of `~/.ssh/id_rsa.pub` file (or other file you used in `user.signingKey` above).
+
+If you're on WSL2, this may help:
```

Comment:
```suggestion
If you’re on **Windows Subsystem for Linux 2**, this may help:
```

1. I replaced `'` symbol to `’` (more correct one even if it looks similar).
2. I explained WSL since not everyone knows the name. Now it is clear that it is related to Windows users.
3. And text highlight